init.lua

```diff
@@ -247,7 +247,10 @@ rtp:prepend(lazypath)
 -- NOTE: Here is where you install your plugins.
 require('lazy').setup({
   -- NOTE: Plugins can be added with a link (or for a github repo: 'owner/repo' link).
-  'NMAC427/guess-indent.nvim', -- Detect tabstop and shiftwidth automatically
+  {
+    'NMAC427/guess-indent.nvim', -- Detect tabstop and shiftwidth automatically
```

Comment:
How about we add this?
```suggestion
    'NMAC427/guess-indent.nvim', -- Detect tabstop and shiftwidth automatically
    event = { 'BufReadPost' },
```

Reply:
Depends, I'd like to know how necessary this is for a pretty small plugin.

I looked at the source, and it looks like if I have to specify loading it on BufReadPost, I'd also have to specify loading it on BufNewFile just to cover the cases where the plugin is intended to work. And will have to watch out for future changes to the plugin's triggers (unlikely).

I wouldn't mind, since the `event` key is already documented and I don't have to do it myself, though I also want to know if it's an issue for the maintainers for a key to be used before its documentation, cause then I'd have to move the code too.

Reply:
Thank you for the answer. I agree that it's overkill to use `event` for such a small plugin.

Reply:
Checked a bit, and the plugin does not load in a simple scenario (nv to empty buffer -> paste some code -> `:w filename` -> continue editing), when BufReadPost and BufNewFile are both specified.
I think it's unnecessary to specify 3 separate events as entry points for a plugin that's attached to a very ubiquitous/common function of the editor (reading existing files/watching for new files). Most people invoke nv with a file to read already, too. Thank you for the suggestion, anyway.